src/dll/Systems/ScriptSystem.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "Hook.hpp"
+#include "ISystem.hpp"
+#include "Paths.hpp"
+#include "PluginBase.hpp"
+
+class ScriptSystem : public ISystem
```

Comment:
I don't think `ScriptSystem` is correct in this case, what about `SccSystem` or `ScriptCompilationSystem`?

Reply:
Would you want to call it something different in the SDK? I think `scripts` is the clearest there, and would just wanna match it in the systems.

Reply:
I think `ScriptSystem` is too generic, from what I see this class is only handling script compilation, right?

Reply:
For now ;) I can change it all to `ScriptCompilationSystem`.